workshop/content/advanced/resource-types/prerequisites/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Prerequisites'
+date: 2021-11-16T20:50:53Z
+weight: 320
+---
+
+### Resource Type Development Tools
```

Comment:
Can we provide example how to install pre-req for maybe one OS. It should be more visible what pre-reqs are needed, maybe as a list with links? Right now there is a lot of text and requirements can be overlooked imo. Some pre-req examples are [here](https://cfn101.workshop.aws/prerequisites.html)

Reply:
Thank you - I will add a list with links for the AWS SAM CLI.